lightning/src/ln/peer_handler.rs

```diff
@@ -930,6 +936,21 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref> PeerManager<D
 					return Err(PeerHandleError{ no_connection_possible: true }.into());
 				}
 			},
+			wire::Message::Warning(msg) => {
+				let mut data_is_printable = true;
+				for b in msg.data.bytes() {
+					if b < 32 || b > 126 {
+						data_is_printable = false;
+						break;
+					}
+				}
```

Comment:
"   - if no existing channel is referred to by `channel_id`:
     - MUST ignore the message."
     
Though dunno if it's intended to apply to `warning` messages and channel_id is also allowed to be blanked out.

Reply:
I raised it on the spec, that's confusing.

Reply:
Rusty noted, and I think I agree, that "ignore" can still mean "log".